<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::WelcomePage</class>
 <widget class="QWidget" name="Core::Internal::WelcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>637</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string>#Core--Internal--WelcomePage {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(247, 247, 247, 255), stop:1 rgba(215, 215, 215, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="gradientWidget" native="true">
     <property name="styleSheet">
      <string>#gradientWidget {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(247, 247, 247, 255), stop:1 rgba(215, 215, 215, 255));
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>57</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>11</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QFrame" name="mainFrame">
        <property name="minimumSize">
         <size>
          <width>662</width>
          <height>515</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>662</width>
          <height>515</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#mainFrame {
    border-image: url(:/core/images/welcomemode/background_center_frame.png) 4;
    border-width: 4;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="headerFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>649</width>
             <height>89</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>649</width>
             <height>89</height>
            </size>
           </property>
           <property name="styleSheet">
            <string>#headerFrame {
    border-image: url(:/core/images/welcomemode/center_frame_header.png) 0;
    border-width: 0;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="navFrame">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string>#navFrame QToolButton {
  border: none; 
  font-size: 10pt;
  color: grey;
}

#navFrame QToolButton:hover {
  color: black;
}

#navFrame QToolButton:checked {
  color: black;
  border-bottom: 1px solid palette(Text);
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QToolButton" name="gettingStartedSectButton">
              <property name="text">
               <string>Getting Started</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="developSectButton">
              <property name="text">
               <string>Develop</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="communitySectButton">
              <property name="text">
               <string>Community</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <widget class="Core::Internal::WelcomeModeButton" name="gettingStartedButton">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>150</y>
               <width>128</width>
               <height>25</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string>* {
    border-image: url(:/core/images/welcomemode/btn_27.png) 7;
    border-width: 7;
    padding: -2px 0;
    font-size: 12px;
    font-family: lucida sans, dejavu sans, sans serif;
    color: black;
}

*:hover {
    border-image: url(:/core/images/welcomemode/btn_27_hover.png) 7;
    color: white;
}
</string>
             </property>
             <property name="text">
              <string>&lt;qt&gt;Getting Started &amp;gt;&amp;gt;</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1" colspan="3">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>578</width>
                 <height>34</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>19</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1">
              <widget class="QFrame" name="recentSessionsFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>270</width>
                 <height>130</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>270</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="iconLabel">
                  <property name="pixmap">
                   <pixmap resource="core.qrc">:/core/images/welcomemode/mode_project.png</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" colspan="2">
                 <widget class="QLabel" name="titleLabel">
                  <property name="text">
                   <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; color:#555555;&quot;&gt;Sessions&lt;/span&gt;&lt;/p&gt;
&lt;hr /&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" colspan="2">
                 <widget class="Core::Internal::WelcomeModeTreeWidget" name="sessTreeWidget">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::NoFocus</enum>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::NoSelection</enum>
                  </property>
                  <property name="verticalScrollMode">
                   <enum>QAbstractItemView::ScrollPerPixel</enum>
                  </property>
                  <property name="rootIsDecorated">
                   <bool>false</bool>
                  </property>
                  <property name="uniformRowHeights">
                   <bool>true</bool>
                  </property>
                  <property name="allColumnsShowFocus">
                   <bool>true</bool>
                  </property>
                  <property name="columnCount">
                   <number>2</number>
                  </property>
                  <attribute name="headerVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="headerDefaultSectionSize">
                   <number>24</number>
                  </attribute>
                  <attribute name="headerMinimumSectionSize">
                   <number>0</number>
                  </attribute>
                  <column>
                   <property name="text">
                    <string notr="true">1</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string notr="true">2</string>
                   </property>
                  </column>
                 </widget>
                </item>
                <item row="2" column="1">
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>58</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="2" column="2">
                 <widget class="Core::Internal::WelcomeModeButton" name="restoreSessionButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string>* {
    border-image: url(:/core/images/welcomemode/btn_26.png) 7;
    border-width: 7;
    padding: -2px 0;
    font-size: 12px;
    font-family: lucida sans, dejavu sans, sans serif;
    color: black;
}

*:hover {
    border-image: url(:/core/images/welcomemode/btn_26_hover.png) 7;
    color: white;
}
</string>
                  </property>
                  <property name="text">
                   <string>&lt;qt&gt;Restore Last Session &amp;gt;&amp;gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>26</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="3">
              <widget class="QFrame" name="recentProjectsFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>270</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="projIconLabel">
                  <property name="pixmap">
                   <pixmap resource="core.qrc">:/core/images/welcomemode/mode_project.png</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="projTitleLabel">
                  <property name="text">
                   <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; color:#555555;&quot;&gt;Projects&lt;/span&gt;&lt;/p&gt;
&lt;hr /&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="Core::Internal::WelcomeModeTreeWidget" name="projTreeWidget">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::NoFocus</enum>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::NoSelection</enum>
                  </property>
                  <property name="verticalScrollMode">
                   <enum>QAbstractItemView::ScrollPerPixel</enum>
                  </property>
                  <property name="rootIsDecorated">
                   <bool>false</bool>
                  </property>
                  <property name="uniformRowHeights">
                   <bool>true</bool>
                  </property>
                  <property name="allColumnsShowFocus">
                   <bool>true</bool>
                  </property>
                  <property name="columnCount">
                   <number>2</number>
                  </property>
                  <attribute name="headerVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="headerDefaultSectionSize">
                   <number>24</number>
                  </attribute>
                  <attribute name="headerMinimumSectionSize">
                   <number>0</number>
                  </attribute>
                  <column>
                   <property name="text">
                    <string notr="true">1</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string notr="true">2</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="4">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0" colspan="4">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>578</width>
                 <height>34</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Seite">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>160</y>
               <width>281</width>
               <height>27</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="helpUsLabel">
                <property name="text">
                 <string>Help us make Qt Creator even better</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Core::Internal::WelcomeModeButton" name="feedbackButton">
                <property name="styleSheet">
                 <string>* {
    border-image: url(:/core/images/welcomemode/btn_26.png) 7;
    border-width: 7;
    padding: -2px 2px;
    font-size: 12px;
    font-family: lucida sans, dejavu sans, sans serif;
}

*:hover {
    border-image: url(:/core/images/welcomemode/btn_26_hover.png) 7;
    color: white;
}
</string>
                </property>
                <property name="text">
                 <string>&lt;qt&gt;Feedback&amp;nbsp;&amp;nbsp;&lt;img src=&quot;:/core/images/welcomemode/feedback_arrow.png&quot; /&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>57</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Core::Internal::WelcomeModeButton</class>
   <extends>QLabel</extends>
   <header>welcomemode_p.h</header>
  </customwidget>
  <customwidget>
   <class>Core::Internal::WelcomeModeTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>welcomemode_p.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="core.qrc"/>
 </resources>
 <connections/>
</ui>
